### DISCUSSION TITLE
Rust installed by pixi VS Code Analyzer

### DISCUSSION THREAD

**Discussion by niko86** [prefix-dev/pixi#1641] [Q&A]
Hello,
I have installed rust using pixi, i dont have rust installed globally. But im hitting problems running the rust-analyzer which i believe allows code completion etc in VS Code. Has anyone added rust in a similar way and got the analyzer in VS Code working?
Thanks

**Comment by Zaubeerer** [comment]
Did you install it with `pixi global install`?
